src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -73,6 +79,12 @@ public static void deleteAllTokens(User user) {
 
   public static String getAccessToken(String userName, String password) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getAccessToken" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)